documentation/docs/configuration.md

```diff
@@ -141,15 +141,18 @@ service-key. The environment variable used is: `PIPER_ansHookServiceKey`.
 
 If Jenkins is used to run "Piper", you can use the Jenkins credential store
 to store the alert notification service service-key as a "Secret Text"
-credential. Provide the credential id in the configuration file in the `hooks`
-section as follows:
+credential. Provide the credential ID in a custom defaults file
+([described below](#custom-default-configuration)) as follows:
 
 ```yaml
 hooks:
   ans:
     serviceKeyCredentialsId: 'my_ANS_Service_Key'
 ```
 
+!!! warning
```

Comment:
warning box: https://stackoverflow.com/questions/50544499/how-to-make-a-styled-markdown-admonition-box-in-a-github-gist